README.md

```diff
@@ -47,10 +47,10 @@ const state = await wn.initialise({
       creator: "Nullsoft"
     },
 
-    // Ask the user permission for additional filesystem paths
+    // Ask the user permission to additional filesystem paths
```

Comment:
doesn't really matter, but I think it should be "Ask the user *for* permission to additional filesystem paths" :slightly_smiling_face:

Reply:
Ah yes that's better 👌